Purchases/Caching/RCInMemoryCachedObject.h

```diff
@@ -0,0 +1,29 @@
+//
+//  RCInMemoryCachedObject.h
+//  Purchases
+//
+//  Created by RevenueCat.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+@interface RCInMemoryCachedObject<ObjectType> : NSObject
+
+- (BOOL)isCacheStale;
+- (void)clearCacheTimestamp;
+- (void)clearCache;
+- (void)updateCacheTimestampWithDate:(NSDate *)date;
+- (void)cacheInstance:(ObjectType)instance
+                 date:(NSDate *)date;
+- (ObjectType _Nullable)cachedInstance;
```

Comment:
There is some inconsistent spacing here

Reply:
could you elaborate? I've been doing the format
```objc
- (returntype)methodWithParameter:(TypeA)parameterName
                   otherParameter:(TypeB)otherParameterName
```
do you mean leaving one empty line between method names?

Reply:
yes, the line between method names